src/sidebar/components/sidebar-panel.js

```diff
@@ -31,8 +31,11 @@ function SidebarPanel({ children, panelName, title }) {
       if (panelIsActive) {
         scrollIntoView(panelElement.current);
       }
+      if (typeof onActiveChanged === 'function') {
```

Comment:
my preference, albit it less robust, is to default the prop to an empty function and not have the condition